test-case/noise_capture_check.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
The  shell script is useful for testing the octave script in isolation but for "real" tests it's probably not needed, existing `test-case/*capture*.sh` tests should probably be modified instead.

Reply:
Yes, that makes sense. Also would need a way to launch this only in devices with plugged in headset and/or analog or digital microphone available. Something like /etc/defaults/has_headset, etc?